tools_docker/Debian_Uwsgi_ARM64/Dockerfile

```
@@ -83,7 +83,7 @@ FROM base
 
 LABEL org.opencontainers.image.authors="contact@tracim.fr"
 
-ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,share_content,upload_permission,gallery"
+ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,share_content,upload_permission,gallery"
```

```suggestion
ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,share_content,upload_permission,gallery"
```
Can you verify that `developement.ini.sample` and `config.py` have todo as default app ?